sdk/core/azure-core/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+`EnvironmentLogLevelListener` (the default logger) now logs all non-error outputs to `stdout` instead of `stderr`. Log level `Error` is still logged to `stderr`.
```

Comment:
nit: `EnvironmentLogLevelListener` is a private class, and I am wondering if we should avoid mentioning it in CL, since it isn't relevant to the user.

Consider excluding it.

```suggestion
The default logger now logs all non-error outputs to `stdout` instead of `stderr`. Log level `Error` is still logged to `stderr`.
```